ui/components/multichain/pages/send/components/account-picker.test.tsx

```diff
@@ -80,4 +92,50 @@ describe('SendPageAccountPicker', () => {
       ).toBeInTheDocument();
     });
   });
+
+  describe('Multichain', () => {
+    it('cannot select BTC accounts', async () => {
+      const mockAccount = createMockInternalAccount();
+      const mockBtcAccount = createMockInternalAccount({
+        name: 'Bitcoin Account',
+        type: BtcAccountType.P2wpkh,
+        address: 'bc1qar0srrr7xfkvy5l643lydnw9re59gtzzwf5mdq',
+      });
+      const { queryByText, queryAllByTestId, getByTestId } = render({
+        metamask: {
+          internalAccounts: {
+            accounts: {
+              [mockAccount.id]: mockAccount,
+              [mockBtcAccount.id]: mockBtcAccount,
+            },
+            selectedAccount: mockAccount.id,
+          },
+          keyrings: [
+            {
+              type: 'HD Key Tree',
+              accounts: [mockAccount.address],
+            },
+            {
+              type: 'Snap Keyring',
+              accounts: [mockBtcAccount.address],
+            },
+          ],
+        },
+      } as CombinedBackgroundAndReduxState);
+
+      expect(queryByText(mockAccount.metadata.name)).toBeInTheDocument();
+
+      const accountPicker = getByTestId('send-page-account-picker');
+      fireEvent.click(accountPicker);
+
+      const addressesInAccountList = queryAllByTestId('account-list-address');
+      expect(addressesInAccountList).toHaveLength(1);
+      expect(addressesInAccountList[0].textContent).toContain(
+        shortenAddress(normalizeSafeAddress(mockAccount.address)),
+      );
+      expect(addressesInAccountList[0].textContent).not.toContain(
+        shortenAddress(normalizeSafeAddress(mockBtcAccount.address)),
+      );
```

Comment:
Nit: A bit more DRY:

```suggestion
      const accountListAddresses = queryAllByTestId('account-list-address');
      expect(accountListAddresses).toHaveLength(1);
      
      const accountListAddressesContent = accountListAddresses[0].textContent;
      expect(accountListAddressesContent).toContain(
        shortenAddress(normalizeSafeAddress(mockAccount.address)),
      );
      expect(accountListAddressesContent).not.toContain(
        shortenAddress(normalizeSafeAddress(mockBtcAccount.address)),
      );
```